stdlib/runtime/bigint.gr

```diff
@@ -136,26 +136,26 @@ let init = (limbs: WasmI32) => {
 provide let debugDumpNumber = (num: WasmI32) => {
```

Comment:
Let's just remove this whole debug function. I really don't like the debug import.

Reply:
Done.